data/com.endlessm.HackToolbox.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.endlessm.HackToolbox.desktop</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>LGPLv2</project_license>
+  <categories>
+    <category>Game</category>
+  </categories>
+  <name>Toolbox</name>
+  <summary>The other side of flip-to-hack</summary>
```

Comment:
I thought there was a "no display" option for the appdata XML as well, isn't there?

Reply:
Scanning through https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html, I don't see one; do you remember what it was called?

Reply:
I see things like this in the appstream published for eos-apps, but I am copying @joaquimrocha here to be sure. Perhaps those are generated by appstream-compose by looking at the desktop file and the appdata.xml at the same time? Who knows!

```
    <vetos>
      <veto>NoDisplay=true</veto>
    </vetos>
```

Reply:
@cosimoc That's true. The `vetos` work, and do get added automagically too. However, this and other similar projects don't get shown directly in g-s in any case because they need an icon (besides the name and summary) in order to be displayed.